extensions.tex

```diff
@@ -79,12 +79,44 @@ \subsection{Beta Distribution Bag-of-words}
   \subsection{Grid Searching}
   \label{sec:grid_search}
 
-    \subsubsection{RNN}
-    \label{sec:rnn_grid_search}
+  \subsubsection{RNN}
+  \label{sec:rnn_grid_search}
 
-\subsection{Ensemble}
-\label{sec:ensemble}
+  \subsection{Ensemble}
+  \label{sec:ensemble}
+  
+  \subsection{Translation}
+  \label{sec:translation}
+ To improve the results of some classifiers, the dataset was enhanced by adding translations of the sentences through an intermediate language back to English. This method was shown to work in a previous Kaggle competition, 'Toxic Comments' \cite{Jigsaw2017}. This is a different application however as it categorises based on the sentiment whereas this challenge is based on the stylistic properties of a given author.
```

Comment:
Several 'this' referring to different things in neighbouring sentences. It would be clearer if some of them were replaced by their full names.

Reply:
I really don't think we should just be saying "yeah we saw this mentioned in the lecture the week before hand in and decided to chuck it in".

If we think translation is worthwhile we should have our own justification for trying it. Not just that we saw another group do it on a different project.